src/net.cpp

```diff
@@ -1642,6 +1642,20 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
     const bool dnsseed = gArgs.GetBoolArg("-dnsseed", DEFAULT_DNSSEED);
     bool add_fixed_seeds = gArgs.GetBoolArg("-fixedseeds", DEFAULT_FIXEDSEEDS);
 
+    // Run the ASMap Health check and log results once every 24h.
+    if (m_netgroupman.UsingASMap() && m_last_asmap_health_check + 24h < NodeClock::now()) {
+        m_last_asmap_health_check = NodeClock::now();
+        const std::vector<CAddress> v4_addrs{GetAddresses(0, 0, Network::NET_IPV4)};
```

Comment:
nit:
```suggestion
        const std::vector<CAddress> v4_addrs{GetAddresses(/*max_addresses=*/0, /*max_pct=*/0, Network::NET_IPV4)};
```

Reply:
done

Reply:
I think you forgot this. (in case you have to touch the code again)

Reply:
Hm, I did do it, not sure why it got lost...